superset/utils/core.py

```diff
@@ -609,15 +610,20 @@ class timeout:  # pylint: disable=invalid-name
     To be used in a ``with`` block and timeout its content.
     """
 
-    def __init__(self, seconds: int = 1, error_message: str = "Timeout") -> None:
+    def __init__(self, seconds: int = 1, error_message: str = "Timeout",) -> None:
```

Comment:
The extra comma may have come from having a different version of `black`.

Reply:
good catch, fixed. thanks!